README.md

```diff
@@ -14,14 +14,14 @@ Ensure you have:
 2. Move into the directory: `cd SeniorDesign2023BE\backend`
 3. Create a Virtual Environment: `python -m venv venv`
 4. Load the venv: `.\venv\Scripts\activate`
-5. Install all the packages: `pip install -r requirements.txt` **IMPORTANT! IF YOU INSTALL OTHER PACKAGES ADD THEM TO "requirements.txt"**
+5. Install all the packages: `pip3 install -r ../requirements.txt --target=".\venv\Lib\site-packages" --upgrade` **IMPORTANT! IF YOU INSTALL OTHER PACKAGES ADD THEM TO "requirements.txt"**
 6. Move into the src directory: `cd src`
 7. Run the server: `flask run` or `python app.py` (running the second command runs it in debug mode, so it will restart every time a file is changed)
 
 #### Other Times:
 1. Move into the main directory: `cd SeniorDesign2023BE\backend`
 2. Load the venv: `.\venv\Scripts\activate`
-3. Ensure you have all the packages: `pip install -r requirements.txt` **IMPORTANT! IF YOU INSTALL OTHER PACKAGES ADD THEM TO "requirements.txt"**
+3. Ensure you have all the packages: `pip3 install -r ../requirements.txt --target=".\venv\Lib\site-packages" --upgrade` **IMPORTANT! IF YOU INSTALL OTHER PACKAGES ADD THEM TO "requirements.txt"**
```

Comment:
../requirements.txt is only needed if youre not in the backend folder. Following the instructions I am already in the backend.